app/assets/javascripts/pageflow/editor/initializers/setup_themes.js

```diff
@@ -0,0 +1,3 @@
+pageflow.app.addInitializer(function(options) {
+  pageflow.editor.themes = new Backbone.Collection(options.themes);
```

Comment:
Same goes for the logic that finds a theme by name, which is also repeated in a couple of places.